### DISCUSSION TITLE
各位没买到烂尾楼的看客们，急啥，保持观望

### DISCUSSION THREAD

**Discussion by mathLjtao** [WeNeedHome/SummaryOfLoanSuspension#1042] [General]
保持观望

**Comment by mathLjtao** [comment]
到时候真正停贷的人，估计都是那些刚需买房的，然后受苦的也是他们。那些炒房客手上有钱，保持观望的态度就行。

**Reply by winnoryder** [reply]
普通老百姓都是受害者，对踩雷的人还是抱有同情，这是人的天性，同情心。只有唯物主义的共产主义垃圾才会幸灾乐祸，但是别忘了梆子总有打在你身上的一天

**Reply by TALK1949** [reply]
不用估计，炒房的也一样，这几年法拍房暴增不是没有原因的。

**Comment by zhaipro** [comment]
偷着乐？那是还没看清什么叫无产阶级命运共同体，并不是我们要有难同当，而是有难必然同当。

**Reply by tanuk1997** [reply]
得了吧，
你以为的无产阶级：有难同单
实际上的无产阶级：哈哈哈~这个傻逼怎么买到烂尾楼的

**Reply by canghaicode** [reply]
一盘散沙

**Comment by volans19** [comment]
大部分人骂共产党，骂那些五毛自干五的时候，并不是想幸灾乐祸，落井下石才去谈政治
更多是抱着恨铁不成钢的心态希望叫醒更多人，虽然希望渺茫但醒一个是一个，他们都是同胞，大家都是正常人，还有正常的同理同情心
如果全都在落井下石的那这个民族就真的没救了，就像你这样的，共产党最喜欢了，底层互害人斗人
而你放心，当你遭到报应的时候，我们不会偷着乐，会在你面前乐

**Reply by mathLjtao** [reply]
所以，你不是那个更乐的人？

**Reply by itsoursduty** [reply]
@mathLjtao  粉红是不是看不懂人话啊？

**Reply by mathLjtao** [reply]
> @mathLjtao 粉红是不是看不懂人话啊？

懂王

**Comment by mathLjtao** [comment]
别被那些甘心为奴的，倒打一耙，还跑去他主人那炫耀

**Comment by ChinaChenMingQuan** [comment]
恶意不买房警告！

**Comment by 505Games-oss** [comment]
总而言之.....不买新房子就对了，看好自己口袋里钱

**Comment by haoqing-yan** [comment]
非必要不买房呗，现在这个政策如此，买房碰上烂尾这辈子都毁了
